content/applications/finance/expenses.rst

```diff
@@ -11,22 +11,30 @@ then be processed and disbursed back to the employee for reimbursement(s).
 .. seealso::
    `Odoo Expenses: product page <https://www.odoo.com/app/expenses>`_
 
-Set expense types
-=================
+Set expense categories
+======================
 
-The first step to track expenses is to configure the different *expense types* for the company
-(managed as *products* in Odoo). Each "product" can be as specific or generalized as needed. Go to
-:menuselection:`Expenses app --> Configuration --> Expense Products` to view the current expensable
-products in a default kanban view.
+The first step to track expenses is to configure the different types of expenses for the company
+(managed as *expense categories* in Odoo). Each category can be as specific or generalized as
+needed. Go to :menuselection:`Expenses app --> Configuration --> Expense Categories` to view the
+current expensable categories in a default list view.
 
-.. image:: expenses/products.png
+.. image:: expenses/categories.png
    :align: center
    :alt: Set expense costs on products.
 
-To create a new expense product, click :guilabel:`Create`. A product form will appear. Only two
-fields are required, the :guilabel:`Product Name` and the :guilabel:`Unit of Measure`. Enter the
-:guilabel:`Product Name` in the field, and select the :guilabel:`Unit of Measure` from the drop-down
-menu (most products will be set to :guilabel:`Units`).
+To create a new expense category , click :guilabel:`New`. A *product* form will appear, with the
+description field labeled :guilabel:`Product Name`.
+
+.. note::
+   Expense *categories* are managed like *products* in Odoo. The expense category form follows the
+   standard *product* form in Odoo, and the information entered is similar. Expense *products* will
+   be referred to as expense *categories* throughout this document since the main menu refers to
+   these as :guilabel:`Expense Categories`.
```

Comment:
OK- for this and the edit above, since the form is a product form, and the name of the "thing" you are creating is a 'category' instead of a 'product' - there was room for confusion, which is why I did all the italics (because it's crazy to call a category a product, or have it function like one but call it the other).